env.js

```diff
@@ -106,14 +107,12 @@ const packagesToRemove = [
   'create-react-class',
 ].map(s => `./node_modules/${s}`);
 
-const additionalDirsToRemove = [
```

Comment:
If these aren’t both removed, then development fails in Travis, and on windows.

Reply:
@ljharb Cool, thanks. I thought it was maybe an artefact of how lerna used to work. I've added them back now.